src/Components/GrotError.tsx

```diff
@@ -1,54 +1,29 @@
 import React from 'react';
 
 import { css } from '@emotion/css';
-import SVG from 'react-inlinesvg';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { useStyles2, useTheme2, Text } from '@grafana/ui';
-
-const getStyles = (theme: GrafanaTheme2) => {
-  return {
-    graphicContainer: css({
-      display: 'flex',
-      justifyContent: 'center',
-      margin: '0 auto',
-    }),
-    graphic: css({ width: '200px', height: '120px', padding: theme.spacing(1) }),
-    text: css({
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center',
-    }),
-    wrap: css({
-      margin: '0 auto',
-    }),
-  };
-};
+import { useStyles2, EmptyState } from '@grafana/ui';
```

Comment:
whoa the EmptyState is animated?? So cool! 
Can we create a follow-up issue to follow up to show the "clear filters" view instead if there are more filters then a single label filter?
<img width="1726" alt="image" src="https://github.com/user-attachments/assets/4933b73f-7709-48f4-b609-a41cd0c788f8" />

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/9c51c03b-b9d9-4f73-97e4-233b1c8657fd" />

Reply:
https://github.com/grafana/logs-drilldown/issues/1164